### PR TITLE
HDS-1070: contributing guide

### PR DESCRIPTION
**Pull request by laitine**
## Description

* CONTRIBUTING.md moved to root
* CONTRIBUTING.md content renewed
* Documentation site contribution guidelines content updated

## Motivation and Context

Users have found it difficult to contribute to the repository and guide texts have divided into multiple places making it difficult to follow. This effort aims to make the content more clear and the process of contributing to HDS more streamlined.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+4. Lint your code. Tip: Lint runs automatically when you build.
+
+```bash
+yarn test:lint
+```
+
+5. Make a Pull Request on the [HDS Github website](https://github.com/City-of-Helsinki/helsinki-design-system/pulls). If you have a ticket number name the PR e.g. "HDS-1377: number input accessibility fix" so it will automatically link to the ticket. Please be sure to check the open PRs in case somebody is already working on a similar issue. Also to prevent overlapping work notify HDS team by Slack at #designsystem or via email hds@hel.fi.
+
+6. HDS team will review the PR and either add it to the release queue, request changes to it or close it with an explanation. PRs are reviewed with the following quality criteria:
+
```
Comment (minevala) [inline CONTRIBUTING.md:32]: Should we also mention something about testing in different screen sizes? We tend to test things on phones, tablets, and large screen sizes. With real devices, simulators, or browser tools

**Review by minevala — COMMENTED** [review]
A really nice guide 👏 👍 
I had a one small question or suggestion

File: site/src/docs/getting-started/contributing/how-to-contribute.mdx [hunk 2]
```diff
@@ -10,16 +10,30 @@ import LeadParagraph from '../../../components/LeadParagraph';
...
 </LeadParagraph>
 
-Every component in HDS includes three parts:
-1. Design 
-2. Implementation 
-3. Documentation 
+Ways to contribute to HDS:
+* Discuss HDS in Slack channel #designsystem (Only for City of Helsinki staff)
+* Attend HDS demo events to get to know new features and ask questions, <InternalLink href="/about/#contact">contact</InternalLink> for invite
+* <InternalLink href="/about/#contact">Contact</InternalLink> HDS team to see how we can help
```
Comment (laurakarhu) [inline site/src/docs/getting-started/contributing/how-to-contribute.mdx:22]: Does the "we" refer to HDS team or the contributors? Maybe contact HDS team to see how we can help each other?

**Review by laurakarhu — APPROVED** [review]
Very good improvements to the guidelines. Well done!
Only one little remark :)

**Pull request merged**